src/GitHub.VisualStudio/Menus/CopyLink.cs

```diff
@@ -45,7 +48,9 @@ public void Activate([AllowNull]object data = null)
 
         public bool CanShow()
         {
-            return IsGitHubRepo();
+            Task<bool> githubRepoCheckTask = IsGitHubRepo();
```

Comment:
You can make this a `var`, the variable name makes it clear it's a task and the method name makes it clear it's a bool.